.circleci/config.yml

```diff
@@ -1,19 +1,14 @@
-version: 2
-jobs:
-  build:
-    machine: true
-    steps:
-    - checkout
+version: 2.1
 
-    - run: |
-        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-        chmod +x ./architect
-        ./architect version
-    - run: ./architect build
-    - store_test_results:
-        path: /tmp/results
-    - deploy:
-        command: |
-          if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            ./architect deploy
-          fi
+orbs:
+  architect: giantswarm/architect@0.8.4
+
+workflows:
+  test:
+    jobs:
+      - architect/go-test:
+          name: go-test
+          filters:
+            # Trigger job also on git tag.
+            tags:
+              only: /^v.*/
```

Comment:
I don't mind but usually, we do it by another PR, don't we?

Reply:
I know. Got excited after figuring out the `go mod tidy` issue.